Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -742,10 +751,12 @@ def relativizePath(String path) {
 class DeployableArtifact {
 	private final String file;
 	private final String deployType;
+	private final Boolean zFSArtifact;
```

Comment:
It's odd to see `zFSArtifact`, I assume if this is false, it assumes to be a PDS member?

What if you introduce a String `objectType` that can be `zFSArtifact`, `PDS-MEMBER`, `SEQ-Dataset` ... So, it allows a bit more flexibility for additional objects, that we want to package in the future.